docs/source/advanced/test-builders.mdx

```diff
@@ -1,38 +1,32 @@
 ---
-title: 'Test Builders'
+title: Test builders
 ---
 
-> Note: Test Builders are experimental and subject to change. We'd love to have your feedback on them. Things that worked well, things that can be improved, etc.. Feel free to reach out either through [Github issues](https://github.com/apollographql/apollo-android/issues/new?assignees=&labels=Type%3A+Bug&template=bug_report.md&title=[Test Builders]) or the [Kotlinlang slack channel](https://app.slack.com/client/T09229ZC6/C01A6KM1SBZ).
```

Comment:
I couldn't access the kotlinlang slack channel from that URL, so I assumed others wouldn't be able to either. Replaced with a link to the forum

Reply:
Maybe the link works only if you're already a member :( That's too bad. We mention this Slack in other places - We could change the link to the register page (https://slack.kotl.in) instead.

Reply:
+1 for keeping the Kotlin slack. Also just bumped into this tweet: https://twitter.com/JimSproch/status/1463108061397061637

During developement and for feedback/discussions about experimental features, slack not being indexed is actually a feature.

Reply:
Perfect, I'll replace with the registration page. Thanks!